.github/workflows/conformance-aks-v1.10.yaml

```diff
@@ -48,9 +48,7 @@ concurrency:
   # Note: for `issue_comment` triggers, we additionally need to filter out based
   # on comment content, otherwise any comment will interrupt workflow runs.
   group: |
-    ${{ github.workflow }}
-    ${{ github.event_name }}
-    ${{
+    ${{ github.workflow }} ${{ github.event_name }} ${{
```

Comment:
Where they not already below 400 characters for GitHub? I checked the GitHub Actions panel in my fork and it wasn't reporting errors anymore :-/

Reply:
I think I might have been misled by the failing actions from PRs that were not rebased on `master`. I realized this while writing the commit description, so I put the PR on draft and am currently testing on my fork. If I indeed got misled I will drop the commit 😅

Reply:
Indeed, confirmed it works and is only due to PR not rebased. Sorry for the confusion, removing second commit :)